config/test/test.config.js

```diff
@@ -21,6 +21,20 @@ const vendorMap = fs
         };
     }, {});
 
+const pkgMap = fs
+    .readdirSync(path.join(root, "packages"))
+    .reduce((map, name) => {
+        const pkgJson = JSON.parse(
+            fs.readFileSync(path.join(root, "packages", name, "package.json")),
+        );
+        return {
+            ...map,
+            // NOTE(kevinb): we the 'source' field here so that we can run our
```

Comment:
"we the 'source'"... is that "we _use_ the 'source'" ?

Reply:
whoopsie